pkg/provider/azure_vmss.go

```diff
@@ -827,12 +827,12 @@ func (ss *ScaleSet) listScaleSetVMs(scaleSetName, resourceGroup string) ([]*armc
 
 	var allVMs []*armcompute.VirtualMachineScaleSetVM
 	var rerr error
-	if ss.Config.VmssVirtualMachineCacheInstanceView {
-		klog.V(4).Info("listScaleSetVMs called for scaleSetName with instanceView: ", scaleSetName, " resourceGroup: ", resourceGroup)
-		allVMs, rerr = ss.ComputeClientFactory.GetVirtualMachineScaleSetVMClient().ListVMInstanceView(ctx, resourceGroup, scaleSetName)
-	} else {
+	if ss.Config.VmssVirtualMachineCacheWithoutInstanceView {
```

Comment:
pls keep the original parameter name

Reply:
What do you mean by the orignial parameter name?

Reply:
Are you refering to `ss.Config.VmssVirtualMachineCacheInstanceView` from my first commit? I like that version better too but this makes it easier to keep the default behavior the same as before and only adds the flag to enable the new option.